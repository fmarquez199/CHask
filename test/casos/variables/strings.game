Runes s1 = ~Hola1~
Runes s2 = ~\*Hola\n1~
Runes s3 = ~Hola2~
Runes s4 = ~Hola2ñ\n~
Runes s4 = ~¡Hey!, ¿Cómo estas?~

