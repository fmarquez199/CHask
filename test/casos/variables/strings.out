Token Runes en la fila: 1, columna: 1
Token identificador "s1" en la fila: 1, columna: 7
Token = en la fila: 1, columna: 10
Token string ~Hola1~ en la fila: 1, columna: 12
Token Runes en la fila: 2, columna: 1
Token identificador "s2" en la fila: 2, columna: 7
Token = en la fila: 2, columna: 10
Token string ~\*Hola\n1~ en la fila: 2, columna: 12
Token Runes en la fila: 3, columna: 1
Token identificador "s3" en la fila: 3, columna: 7
Token = en la fila: 3, columna: 10
Token string ~Hola2~ en la fila: 3, columna: 12
Token Runes en la fila: 4, columna: 1
Token identificador "s4" en la fila: 4, columna: 7
Token = en la fila: 4, columna: 10
Token string ~Hola2ñ\n~ en la fila: 4, columna: 12
Token Runes en la fila: 5, columna: 1
Token identificador "s4" en la fila: 5, columna: 7
Token = en la fila: 5, columna: 10
Token string ~¡Hey!, ¿Cómo estas?~ en la fila: 5, columna: 12
